### PR TITLE
Add issueType to jira board schema

### PR DESCRIPTION
**Pull request by ralphbean**
This permits a team to specify a new issue type for jiralert to use.

See downstream: https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/76908

Issue https://issues.redhat.com/browse/RHTAPSRE-129

### CODE REVIEW

File: schemas/dependencies/jira-board-1.yml [hunk 1]
```diff
@@ -31,6 +31,13 @@ properties:
         type: string
     required:
     - workspace
+  issueType:
+    type: string
+    enum:
+    - Task
+    - Bug
+    - Ticket
+    - Incident
```
Comment (geoberle) [inline schemas/dependencies/jira-board-1.yml:40]: i understand the idea of having a sanity check for issue types at the schema level. afaik issue types are jira instance and project specific (e.g. ticket and incident is not something i can use in our team project). hence we can't rely on this to make sure an alert will be created successfully on a specific board. 

i suggest to remove to enum

Comment (ralphbean) [reply schemas/dependencies/jira-board-1.yml:40]: Makes sense. Will drop it.

Comment (ralphbean) [reply schemas/dependencies/jira-board-1.yml]: Done.

Comment (geoberle) [reply schemas/dependencies/jira-board-1.yml]: thanks

**Review by geoberle — APPROVED** [review]
LGTM. thanks for your contribution

**Pull request merged**